biorbd_optim/misc/optimal_control_program.py

```diff
@@ -615,6 +597,94 @@ def update_parameters(self, new_parameters):
         else:
             raise RuntimeError("new_parameter must be a ParameterOption or a ParameterList")
 
+    def update_bounds(self, X_bounds=None, U_bounds=None):
+        if X_bounds is not None:
+            self.def_X_bounds = True
```

Comment:
The default should be an empty bound list (`X_bounds=BoundsList()`) but ignored if is empty (either test if `len > 0` or have a proper test by overloading `__ bool __` which will allow for `if X_bounds`)
[Same is true for U_bounds]